stable/mariadb/values.yaml

```diff
@@ -64,16 +64,16 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  # storageClass: "-"
+  storageClass: "-"
```

Comment:
this should be `storageClass: ` without `"-"` as the default. Setting `"-"` will disable dynamic provisioning which is causing the test to fail.

Reply:
DOne